multigeiger/speaker.cpp

```diff
@@ -65,11 +65,11 @@ void IRAM_ATTR isr_audio() {
     led = *p++;
     duration_ms = *p++;
     isr_sequence = p;
```

Comment:
@ThomasWaldmann: 1) Here we defined the frequency, volume, led and duration for the last sequence. The values are valid, as isr_sequence was not zero, so their interpretation in the second part of the function should be carried out. If duration is now 0, it's the last part of the sequence and p (=isr_sequence) is NULL.

Reply:
if will only execute 65+ if p is not NULL.
if duration is 0, it will **later** be set p to NULL, but that is inside a mutex block.

Reply:
Right. but this later will not happen for the last block, as p will be NULL then and an 'if (!p)' will return.